CHANGELOG.md

```diff
@@ -114,6 +114,17 @@ In the future, the component will automatically use the use the `<a>` element if
 
 This change was introduced in [pull request #4646: Deprecate `element` parameter on button component](https://github.com/alphagov/govuk-frontend/pull/4646)
 
+#### Stop using `govuk-body-xs`, `govuk-!-font-size-14` and '14' on the typography scale
+
+We have deprecated 14 on the responsive typography scale as well as classes within GOV.UK Frontend which set text to 14px:
+
+- `govuk-body-xs`
+- `govuk-!-font-size-14`
+
+We will be removing these classes and 14 on the typography scale in a future major release (6.0.0). From the next major release, you will no longer be able to call the Sass mixins `govuk-font` or `govuk-font-size` with `$size` set to '14'.
+
+This change was introduced in [#4649: Deprecate 14 on the typography scale](https://github.com/alphagov/govuk-frontend/pull/4649)
```

Comment:
I'm not suggesting we change the 'typography scale' here because that would potentially mean changing the PR title as well.

Reply:
I changed the PR title anyway 😎

Reply:
Ah ok!